frontend/src/metabase/admin/settings/components/Email/SMTPConnectionForm.tsx

```diff
@@ -113,50 +139,140 @@ export const SMTPConnectionForm = ({
+                }}
+                descriptionProps={{
+                  fz: "0.75rem",
+                  mb: "0.5rem",
+                }}
+              />
+              <FormRadioGroup
+                name="email-smtp-security"
+                label={elementMap["email-smtp-security"]["display_name"]}
+                description={elementMap["email-smtp-security"]["description"]}
+                mb="1.5rem"
+                labelProps={{
+                  tt: "uppercase",
+                  fz: "0.875rem",
+                  c: "text-medium",
+                  mb: "0.5rem",
+                }}
               >
-                {t`Clear`}
-              </Button>
-            </>
+                <Group>
+                  {Object.entries(
+                    elementMap["email-smtp-security"].options || {},
+                  ).map(([value, name]) => (
+                    <Radio
+                      value={value}
+                      label={name}
+                      key={value}
+                      styles={{
+                        inner: { display: "none" },
+                        label: {
+                          paddingLeft: 0,
+                          color:
+                            values["email-smtp-security"] === value
+                              ? color("brand")
+                              : color("text-dark"),
+                        },
+                      }}
+                    />
+                  ))}
+                </Group>
+              </FormRadioGroup>
+              <FormTextInput
+                name="email-smtp-username"
+                label={elementMap["email-smtp-username"]["display_name"]}
+                description={elementMap["email-smtp-username"]["description"]}
+                placeholder={elementMap["email-smtp-username"]["placeholder"]}
+                mb="1.5rem"
+                labelProps={{
+                  tt: "uppercase",
+                  mb: "0.5rem",
+                }}
+              />
+              <FormTextInput
+                name="email-smtp-password"
+                type="password"
+                label={elementMap["email-smtp-password"]["display_name"]}
+                description={elementMap["email-smtp-password"]["description"]}
+                placeholder={elementMap["email-smtp-password"]["placeholder"]}
+                mb="1.5rem"
+                labelProps={{
+                  tt: "uppercase",
+                  mb: "0.5rem",
+                }}
+              />
+              {testEmailError && (
+                <Text
+                  role="alert"
+                  aria-label={testEmailError}
+                  color="error"
+                  mb="1rem"
+                >
+                  {testEmailError}
+                </Text>
+              )}
+              <Box mt="1rem">
+                <FormSubmitButton
+                  label={t`Save changes`}
+                  disabled={!dirty}
+                  mr="1.5rem"
```

Comment:
marginInlineEnd could go here I think

Reply:
Or perhaps it's neater to use flexbox

Reply:
I briefly threw them in a flex box but it wanted to put them on 2 lines haha. I could try again though (I didn't try very hard)